### PR TITLE
build: add an ignore list to the generator

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Sometimes there are APIs that are just broken, like apigee:v1 right now.  Creating a lightweight ignore list so we can opt these out.

### CODE REVIEW

File: src/generator/generator.ts [hunk 1]
```diff
@@ -232,14 +232,19 @@ export class Generator {
...
     const res = await this.request<Schemas>({url: discoveryUrl, headers});
     const apis = res.data.items;
     const queue = new Q({concurrency: 10});
     console.log(`Generating ${apis.length} APIs...`);
     queue.addAll(
       apis.map((api: {[key: string]: string}) => {
         return async () => {
-          this.log('Generating API for %s...', api.id);
+          // look at ignore.json to find a list of APIs to ignore
+          if (ignore.indexOf(api.id) >= 0) {
```
Comment (alexander-fenster) [inline src/generator/generator.ts:244]: we can now use `includes`? :)

File: src/generator/generator.ts [hunk 2]
```diff
@@ -232,14 +232,19 @@ export class Generator {
...
+            return;
```
Comment (alexander-fenster) [inline src/generator/generator.ts:245]: Log message will be helpful. AFAIR the generator prints APIs it generates, so makes sense to print APIs it ignores as well.

File: src/generator/generator.ts (hunk 1 above)
Comment (JustinBeckwith) [reply src/generator/generator.ts]: You're right!

File: src/generator/generator.ts (hunk 2 above)
Comment (JustinBeckwith) [reply src/generator/generator.ts:246]: Good call!

**Pull request merged**